SQL/0000-00-03-ConfigTables.sql

```diff
@@ -133,7 +130,7 @@ INSERT INTO ConfigSettings (Name, Description, Visible, AllowMultiple, DataType,
 
 -- Loris-MRI/Imaging Pipeline options from the $profile (commonly "prod") file
 INSERT INTO ConfigSettings (Name, Description, Visible, AllowMultiple, Label, OrderNumber) VALUES ('imaging_pipeline', 'Imaging Pipeline settings', 1, 0, 'Imaging Pipeline', 12);
-INSERT INTO ConfigSettings (Name, Description, Visible, AllowMultiple, DataType, Parent, Label, OrderNumber) SELECT 'dataDirBasepath', 'Base Path to the data directory of Loris-MRI', 1, 0, 'text', ID, 'Loris-MRI Data Directory', 1 FROM ConfigSettings WHERE Name="imaging_pipeline";
+INSERT INTO ConfigSettings (Name, Description, Visible, AllowMultiple, DataType, Parent, Label, OrderNumber) SELECT 'dataDirBasepath', 'Base Path to the data ' || 'directory of Loris-MRI', 1, 0, 'text', ID, 'Loris-MRI Data Directory', 1 FROM ConfigSettings WHERE Name="Paths";
```

Comment:
The description of this should specify whether this is relative to the imaging server or the web server (and the type will need to be changed once #4023 is merged)

Reply:
At the last imaging meeting, we discussed we would only keep one setting so technically this one would work for both. We can bring it up again at the next imaging meeting. I'll add it to the agenda.
Thanks!

Reply:
But it can't work for both imaging pipeline and web server, the paths are different if they're mounted on different places on the different servers

Reply:
@driusan in the next commit I reintegrated the imagePath for the apache server and kept the dataDirBasepath for the LORIS-MRI server. Please re-review